CMakeLists.txt

```diff
@@ -30,6 +39,11 @@ catkin_package(
     DEPENDS Boost Eigen OMPL OpenRAVE
 )
 
+if(OR_OMPL_HAS_BOOSTSMARTPTRS)
+else(OR_OMPL_HAS_BOOSTSMARTPTRS)
+add_definitions(-std=c++11)
```

Comment:
You should only use `add_definitions` for `-D` flags. Please add this to `CMAKE_CXX_FLAGS` instead.